gno.land/pkg/gnoweb/handler.go

```diff
@@ -420,6 +420,28 @@ func (h *WebHandler) GetHelpView(gnourl *weburl.GnoURL) (int, *components.View)
 	})
 }
 
+// renderReadme renders the README.md file and returns the component and the raw content
+func (h *WebHandler) renderReadme(gnourl *weburl.GnoURL, pkgPath string) (components.Component, []byte) {
```

Comment:
Maybe a crazy question, but what happens if I have an insanely big `README.md`?
Should we guard against this somehow? I don't think we pay any additional gas for `md` files pushed to the chain

cc @ajnavarro

Reply:
We definitely pay more gas for md files pushed to the chain

They write more data to the underlying DB, so they're automatically counted.

Maybe not enough, but there is some limit to how much can be written